core/src/components/textarea/test/a11y/textarea.e2e.ts

```diff
@@ -0,0 +1,17 @@
+import AxeBuilder from '@axe-core/playwright';
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('textarea: a11y', () => {
+  test.beforeEach(async ({ skip }) => {
+    skip.rtl();
+    skip.mode('md');
```

Comment:
The rendering of the inputs will vary based on platform so there may be axe violations that we miss here. I should not have done this in my `ion-input` changes so I will remove that from the input a11y test.